src/main/java/core/mate/academy/service/producer/TruckProducer.java

```diff
@@ -0,0 +1,16 @@
+package core.mate.academy.service.producer;
+
+import core.mate.academy.model.Truck;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TruckProducer implements MachineProducer<Truck> {
+    @Override
+    public List<Truck> get() {
+        List<Truck> trucks = new ArrayList<>();
+        trucks.add(new Truck());
+        trucks.add(new Truck());
+        trucks.add(new Truck());
```

Comment:
Use the new constructor I told you to create in models (Do it in all `MachineProducer` implementations).